pyuvdata/uvflag.py

```diff
@@ -22,19 +22,29 @@ class UVFlag(object):
 
     def __init__(self, input, mode='metric', copy_flags=False, waterfall=False, history='',
                  label=''):
-        '''Initialize UVFlag object.
-        Args:
-            input: UVData object, UVCal object, or path to previously saved UVFlag object.
-                   Can also be a list of any compatible combination of the above options.
-                   UVData and UVCal objects cannot be directly combined, unless waterfall is True.
-            mode: "metric" (default) or "flag" to initialize UVFlag in given mode.
-                  The mode determines whether the object has a floating point metric_array
-                  or a boolean flag_array.
-            copy_flags: Whether to copy flags from input to new UVFlag object. Default is False.
-            waterfall: Whether to immediately initialize as a waterfall object,
-                       with flag/metric axes: time, frequency, polarization. Default is False.
-            history: History string to attach to object.
-            label: string used for labeling the object (e.g. 'FM')
+        '''Initialize UVFlag object. Metadata is copied from input object. If input
```

Comment:
maybe not a big deal, but numpy docstyle does specify triple *double* quotes (not single quotes). https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard